pyproject.toml

```diff
@@ -63,6 +63,9 @@ docs = [
 discord = ["discord-disnake"]
 
 [tool.pdm.dev-dependencies]
+nox = [
+    "nox>=2022.11.21",
```

Comment:
maybe consider using `==` instead? There could always be breaking changes, and since it's CalVer we can't really use `~=` here either :/

Reply:
There always could be breaking changes, but this should only be installed in local development, so, its only pinning our own developers.

Reply:
Well, its now installed for pyright typechecking, so I suppose you're not wrong.

Reply:
> but this should only be installed in local development, so, its only pinning our own developers.

That's fair. But yeah, if we use it for type-checking, pinning it would be great